docs/pages/api-reference/client.md

```diff
@@ -83,8 +83,8 @@ This api is an asynchronous api that returns a request id and the path to the ou
 **Arguments:**
 
 
-* `input_feature_list: List[Union[Feature, Featureset]]` - List of features or featuresets to use as input.
-* `output_feature_list: List[Union[Feature, Featureset]]` - List of features or featuresets to compute.
+* `inputs: List[Union[Feature, Featureset]]` - List of features or featuresets to use as input.
+* `outputs: List[Union[Feature, Featureset]]` - List of features or featuresets to compute.
```

Comment:
Remove Featureset from extract historical too, please do this for client.py too

Reply:
Also add str to the type ( ensure the client.py and docs are synced )

Reply:
yes